src/main/java/ResponseBuilder.java

```diff
@@ -1,9 +1,21 @@
 public class ResponseBuilder {
+
     private static final String EMPTY_BODY_OUTPUT = "";
-    private static final String STATUS_CODE_OK = "HTTP/1.1 200 OK\r\n";
-    private static final String OUTPUT_END_OF_HEADERS = "\r\n";
+    private static final String STATUS_CODE_OK = "HTTP/1.1 200 OK";
+    private static final String CRLF = "\r\n";
+    private static final String ALLOWED_HEADER = "Allow: ";
+    private static final String GET_HEAD_OPTIONS = "GET, HEAD, OPTIONS";
+    private static final String PUT_AND_POST = ", PUT, POST";
 
     public String okayWithEmptyBody() {
-        return STATUS_CODE_OK + OUTPUT_END_OF_HEADERS + EMPTY_BODY_OUTPUT;
+        return STATUS_CODE_OK + CRLF + CRLF + EMPTY_BODY_OUTPUT;
+    }
+
+    public String okayWithHeadersGetHeadOptions() {
```

Comment:
These methods feel very specific. I really like the idea of a builder class for the response but consider whether you could make it a more re-usable component. The builder pattern might be worth exploring